st2common/st2common/openapi.yaml

```diff
@@ -1160,10 +1160,26 @@ paths:
           required: true
         - name: output_type
           in: query
-          description: Type of output to retrieve. If not provided, all output type is returned.
-          type: array
-          items:
-            type: string
+          description: Type of output to retrieve (stdout, stderr). If not provided, all output type is returned.
+          type: enum
```

Comment:
there's no `type: enum` in the spec. It's `type: string` with `enum` as property.

Reply:
Thanks, will fix that. I would assume it would error our on invalid type.